wikimedia/wikipedia-ios
# Edit summary design review feedback

**Phabricator:** https://phabricator.wikimedia.org/T354219

### Notes
Tweaks margins and divs

### Test Steps
1. Go to Edit Summary view on a Test Wiki article.
2. Confirm margins and divs look okay.
